compiler/damlc/daml-stdlib-src/DA/Internal/LF.daml

```diff
@@ -90,8 +90,8 @@ partyToText = primitive @"BEToText"
 -- valid party strings and parties. In other words, the following equations hold:
 --
 -- ```
--- ∀ p. partyFromText (partyToText p) = Some p
--- ∀ txt p. partyFromText txt = Some p ==> partyToText p = txt
+-- forall p. partyFromText (partyToText p) = Some p
+-- forall txt p. partyFromText txt = Some p ==> partyToText p = txt
```

Comment:
Is this Sphinx-related? Is the new Sphinx banning non-ascii characters or something?

Reply:
Or is that a LaTeX issue, perhaps?

Reply:
sphinx now errors out on syntax errors. You can override it with `:force:` which I’ve done for some issues in our typescript code but here changing our code seems easier.

Reply:
I switched things around since there were other places that blew up and now you can tag code blocks and daml-docs emits :force: annotations.